packages/components/src/components/action/action.tsx

```diff
@@ -81,9 +81,12 @@ export class Action extends LitElement implements FormOwner {
       | "labelledByElements"
       | "ownsElements"
       | "pressed"
+      | "checked"
     >
   >;
 
+  @property({ reflect: true }) role: LuminaJsx.AriaAttributes["role"] = "button";
```

Comment:
This is overriding [`Element#role`](https://developer.mozilla.org/en-US/docs/Web/API/Element/role). All components extend `HTMLElement`, which extends `Element`. You can access the host's `role` by going the element (e.g., `this.el.role`).